app/models/facilities_query/zip_query.rb

```diff
@@ -5,10 +5,10 @@ class ZipQuery < Base
     def run
       # TODO: allow user to set distance from zip
       zip_plus0 = params[:zip][0...5]
-      requested_zip = ZCTA.select { |area| area[0] == zip_plus0 }
+      requested_zip = ZCTA[zip_plus0]
```

Comment:
I'm not even sure this class is being used anymore, but changed it to reflect that ZCTA is now a map.